.storybook/themes/news.scss

```diff
@@ -812,6 +812,53 @@
 					list-style-position: inside,
 					list-style-type: disc,
 				),
+				author-bio: (
+					gap: var(--global-spacing-4),
+				),
+				author-bio-author: (
+					gap: var(--global-spacing-4),
+					display: flex,
+					flex-direction: row,
+					components: (
+						image: (
+							max-width: var(--global-spacing-12),
+							max-height: var(--global-spacing-12),
+							border-radius: var(--global-border-radius-2),
+						),
+						stack: (
+							flex: 1,
+						),
+					),
+				),
+				author-bio-social-link-wrapper: (
+					display: flex,
+					flex-wrap: wrap,
+					components: (
+						link: (
+							margin: var(--global-spacing-2) var(--global-spacing-4) 0 0,
+						),
+					),
+				),
+				author-bio-social-link-hover: (
+					components: (
+						icon: (
+							fill: var(--color-primary-hover),
+						),
+					),
+				),
+				author-bio-authorName: (
+					font-family: var(--font-family-primary),
+					font-size: var(--global-font-size-6),
+					font-weight: var(--global-font-weight-7),
+					margin-bottom: var(--global-spacing-2),
+					line-height: var(--global-line-height-5),
+				),
+				author-bio-authorDescription: (
+					font-size: var(--global-font-size-3),
```

Comment:
font-size and font-weight should be using an alias token, if possible. have you and @haganka talked about using different values from our alias tokens. 

We can talk over alias tokens more in depth and review that if that's helpful

Reply:
Should be `body-font-size-small` and `body-line-height-small`  (be sure to reset to `body-font-size` and `body-line-height` on desktop)

Reply:
Fixed with alias-tokens

Reply:
addressed

Reply:
fixed -- nice

Reply:
was this updated on desktop as well? @malavikakoppula I don't see line height change on desktop

https://github.com/WPMedia/arc-themes-blocks/pull/1474#discussion_r955451976